src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const objectOfStyles = {};
+  const stylesArray = sourceString.split(';');
+
+  for (let style of stylesArray) {
+    if (style.trim().length < 3) {
+      continue;
+    }
+
+    style = style.split(':').map((s) => s.trim());
```

Comment:
Consider adding a check to ensure that the `style` array contains exactly two elements after splitting. This will prevent errors if the input string contains malformed styles without a colon or with multiple colons.